### PR TITLE
chor: cleanup & simplify cli commands

### PR DESCRIPTION
**Pull request by xalisys**
cleanup & simplify cli commands

### CODE REVIEW

**Comment by xalisys** [comment]
done, i added .pretterrc as well to auto format with vscode

**Comment by xalisys** [comment]
sure, i'll update.

File: packages/react-google-maps-api-storybook/src/GoogleMap.stories.js [hunk 1]
```diff
@@ -2,109 +2,97 @@
...
+import { LoadScript, GoogleMap, StreetViewPanorama } from '@react-google-maps/api'
+import googleMapKey from '../googleMapKey'
 
 const isBrowser = typeof document !== 'undefined'
 
 const mapContainerStyle = {
-  height: '700px',
-  width: '100%'
+    height: '700px',
+    width: '100%'
```
Comment (JustFly1984) [inline packages/react-google-maps-api-storybook/src/GoogleMap.stories.js]: please fix indentation

Comment (JustFly1984) [reply packages/react-google-maps-api-storybook/src/GoogleMap.stories.js]: PS. messing with indentation ruins `git blame` functionality

**Pull request merged**